### PR TITLE
Bump golang to v1.14

### PR DESCRIPTION
**Pull request by tibkiss**
This PR also addresses the module loading issue discovered during polyfeed testing.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 # Uses a Go image to build a release binary.
 #
-FROM golang:1.13.0-buster as builder
+FROM golang:1.14.0-buster as builder
```
Comment (leki75) [inline Dockerfile:6]: I would use `golang:1.14.2-buster` as that is the latest go version. `golang:1.14-buster` seems to be even better if we do not want to hardcode the exact version.

File: Dockerfile [hunk 2]
```diff
@@ -17,11 +17,9 @@ RUN if [ "$INCLUDE_PLUGINS" = "true" ] ; then make build plugins ; else make bui
 #
 # STAGE 2
 #
-# Use a tiny base image (alpine) and copy in the release target. This produces
-# a very small output image for deployment.
+# Create final image
 #
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates tzdata libc6-compat
+FROM debian:buster
```
Comment (leki75) [inline Dockerfile:22]: This should be `debian:10.3` if we stick to exact versions, however the image dependency for `golang:1.14.2-buster` looks: `golang:1.14.2-buster` -> `buildpack-deps:buster-scm` -> `buildpack-deps:buster-curl` -> `debian:buster`.

Comment (dakimura) [inline Dockerfile:22]: LGTM but could you tell me the context of the docker image change? 👀

Comment (tibkiss) [reply Dockerfile:22]: yep, the alpine -> debian change is required to be able to load .so files at runtime.
Without this change marketstore hanged when tried to load modules, e.g.:
```{"level":"info","timestamp":"2020-04-22T08:14:39.024Z","msg":"Trying to load module from path: /bin/polygon.so...\n"}
{"level":"info","timestamp":"2020-04-22T08:14:39.069Z","msg":"Success loading module /bin/polygon.so.\n"}
```

File: Dockerfile (hunk 1 above)
Comment (tibkiss) [reply Dockerfile:6]: thanks, good point!

File: Dockerfile (hunk 2 above)
Comment (tibkiss) [reply Dockerfile:22]: given that buster moves I'll change this to debian:10.3

**Pull request merged**